.changeset/light-horses-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/theme': patch
+---
+
+I've changed the link color in dark theme to #EFE5B1 instead dark blue
```

Comment:
Rather than using the personal "I've changed" and putting the color here, this should be more generic. Something more like what you have in the title of the PR: "Change link color in dark theme"?

Reply:
@adamdmharvey Done